internal/pkg/utils/utils.go

```diff
@@ -614,3 +619,50 @@ func ResolvePools(ctx context.Context, c client.Interface, pools []string, isv4
 	}
 	return result, nil
 }
+
+func AddNode(c client.Interface, kc *kubernetes.Clientset, host string) error {
```

Comment:
These should go in the `internal/pkg/testutils` package since they are just meant for test code.